packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -254,4 +254,21 @@ export interface ISemanticColors {
    * The background color of a checked and hovered list item.
    */
   listItemBackgroundCheckedHovered: string;
+
+  //// Rating
```

Comment:
Semantic slots should never be component-specific. They should be usable across a broad range of components.

ratingDefaultColor can use bodySubtext
ratingSelected can use bodyText or bodyTextChecked
ratingDisabled can use disabledBodyText